docs/setup-vendor.md

```diff
@@ -0,0 +1,47 @@
+# Vendor's Customers using Private Packagist in a Composer project
```

Comment:
```suggestion
# Private Packagist for Vendors: Using Private Packages in Customer Projects
```

Reply:
This has not been applied yet?

Reply:
ok applied now and double checked the rest.